java/test/jmri/jmrit/display/layoutEditor/LayoutEditorEditorTest.java

```diff
@@ -0,0 +1,348 @@
+package jmri.jmrit.display.layoutEditor;
+
+import java.awt.Component;
+import java.awt.Container;
+import java.awt.GraphicsEnvironment;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JDialog;
+import javax.swing.JMenuBar;
+import javax.swing.MenuElement;
+import jmri.Turnout;
+import jmri.util.JUnitUtil;
+import jmri.util.JmriJFrame;
+import jmri.util.MathUtil;
+import junit.extensions.jfcunit.TestHelper;
```

Comment:
Drop all the jfcunit imports.  JFCUnit is no longer being maintained, and we are moving on to Jemmy.

Reply:
Done.

Reply:
I still see both jfcunit and Jemmy imports here....

Reply:
Offending file removed (I checked; I'd already removed it from PackageTest.java)